akka-http-bench-jmh/src/main/scala/akka/http/impl/engine/HeaderParserBenchmark.scala

```diff
@@ -58,6 +58,7 @@ class HeaderParserBenchmark {
   @Benchmark
   def bench_parse_headers(): Unit = {
     val next = parser.parseHeaderLine(requestBytes, firstHeaderStart)()
-    parser.parseHeaderLine(requestBytes, next)()
+    val last = parser.parseHeaderLine(requestBytes, next)()
+    require(last == 84, s"Expected to read until 84, but read until $last")
```

Comment:
this is still incorrect, you have to return the value (then it is blackholed) or blackhole it by using an explicit one.

Refer to https://github.com/ktoso/sbt-jmh/blob/d20ad9145da005deba2442055cdb53216c339e56/plugin/src/sbt-test/sbt-jmh/run/src/main/scala/org/openjdk/jmh/samples/JMHSample_09_Blackholes.scala#L85 

`measureRight*` vs `measureWrong`

Reply:
You can also skim http://java-performance.info/jmh/ which is a softer introduction